src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -189,7 +189,7 @@ Copyright (c) .NET Foundation. All rights reserved.
                               TargetingPackVersion="$(NetCoreAppTargetingPackVersion)"
                               AppHostPackNamePattern="runtime.**RID**.Microsoft.NETCore.DotNetAppHost"
```

Comment:
Noticed this... Core-Setup has produced `Microsoft.NETCore.App.Host.<RID>` for a while, just want to make sure there aren't any blocking issues I'm not aware of.

Reply:
Ah, it was unused! I was frightened by this comment until I saw da3a92a

Reply:
And I was frightened by your comment until I saw that `Microsoft.NETCore.App.Host.<RID>` is used, just by a different item that doesn't show up in the PR view without a little expanding:

https://github.com/dotnet/core-sdk/blob/5293e96d5a019b41930c1c38396f21d13c207561/src/redist/targets/GenerateBundledVersions.targets#L196-L201

😄